src/components/content/Form.js

```diff
@@ -1,26 +1,65 @@
 import React, {Component} from 'react';
 import "bootstrap/dist/css/bootstrap.css";
+
 class Form extends Component {
+    handleSubmit = (event) => {
+        event.preventDefault();
+        const email = document.getElementById('inputEmail');
+        const pass = document.getElementById('inputPassword');
+        const rememberCheckBox = document.getElementById('check').getAttribute('checked');
+        const url = 'https://5b3643ec6005b00014c5dcf2.mockapi.io/users';
```

Comment:
It is wrong using this variables. It seems like local state or jsut as a constant.